django/db/models/constraints.py

```diff
@@ -16,10 +16,15 @@
 
 class BaseConstraint:
     default_violation_error_message = _("Constraint “%(name)s” is violated.")
+    violation_error_code = None
     violation_error_message = None
 
-    def __init__(self, name, violation_error_message=None):
+    def __init__(
+        self, name, violation_error_message=None, *, violation_error_code=None
```

Comment:
I think we should change the signature of `BaseConstraint` to use keyword-only arguments as a separate cleanup. Preferably, before merging this patch. Would you like to prepare PR?

Reply:
:laughing: I was actually considering making a follow-up PR with this change but sure I could also do it before.
No need for a deprecation period/message ?

Reply:
Cf https://github.com/django/django/pull/16578